tests/src/image.rs

```diff
@@ -458,20 +472,37 @@ pub struct ReadbackBuffers {
 impl ReadbackBuffers {
     pub fn new(device: &Device, texture: &Texture) -> Self {
         let (block_width, block_height) = texture.format().block_dimensions();
-        let base_size = (texture.width() / block_width)
-            * (texture.height() / block_height)
-            * texture.depth_or_array_layers();
+        let should_align_buffer_size = !([
+            TextureFormat::Depth24Plus,
+            TextureFormat::Depth24PlusStencil8,
```

Comment:
I don't understand why we're only aligning to the buffer size on these types. No matter the texture format, we need to align the copies to the bytes_per_row alignment.

Reply:
It is the opposite, we are **not aligning** if the texture format is one of these. The reason why we are not checking the alignment is that we are using a custom compute pipeline to copy the data if the depth format is Depth24Plus. And as far as I know, that is because as of [webgpu spec](https://gpuweb.github.io/gpuweb/#depth-formats), depth aspects of textures with those formats are an invalid copy source.

Reply:
Improved the readability of these lines too